src/engine/Marabou.cpp

```diff
@@ -93,6 +93,12 @@ void Marabou::prepareInputQuery()
         */
         String networkFilePath = Options::get()->getString( Options::INPUT_FILE_PATH );
 
+        if ( networkFilePath.length() == 0 )
+        {
+            printf( "Error: no network file provided!\n" );
+            throw MarabouError( MarabouError::FILE_DOESNT_EXIST, networkFilePath.ascii() );
+        }
+
         if ( !File::exists( networkFilePath ) )
```

Comment:
Change this line to "else if" then?

Reply:
I'd prefer not to. I think early return points are much cleaner to read and require less mental effort. However, if you insist I can change it?